cmd/mistryd/job.go

```diff
@@ -275,3 +278,51 @@ func GetState(path, project, id string) (string, error) {
 	}
 	return "", fmt.Errorf("job with id=%s not found error", id)
 }
+
+func GetJobs(path string) ([]Job, error) {
```

Comment:
Add documentation. Also, this should be unexported.

Reply:
This belongs to the server.

Reply:
Done